src/axolotl/utils/models.py

```diff
@@ -70,8 +74,12 @@
 LOG = logging.getLogger(__name__)
 
 MULTIMODEL_AUTO_MODEL_MAPPING = {
```

Comment:
Just realized there's a typo here.

```suggestion
MULTIMODAL_AUTO_MODEL_MAPPING = {
```